climpred/bootstrap.py

```diff
@@ -316,7 +318,8 @@ def bootstrap_compute(
         hist (xr.Dataset): historical/uninitialized simulation.
         metric (str): `metric`. Defaults to 'pearson_r'.
         comparison (str): `comparison`. Defaults to 'm2e'.
-        dim (str or list): dimension to apply metric over. default: 'init'
+        dim (str or list): dimension to apply metric over. default: 'init'.
+        resample_dim (str or list): dimension to resample from. default: 'member'.
```

Comment:
Could you add a note/bullet list here explaining the difference between resampling over 'member' and 'init'? Or at least explain to me here what the differences are. I don't think I understand in which cases you want each dimension.

Reply:
should be only str. added example